packages/odyssey-react/src/components/Checkbox/Checkbox.theme.ts

```diff
@@ -13,34 +13,40 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  ColorBorder: theme.ColorBorderUi,
-  ColorBorderChecked: theme.ColorPrimaryBase,
-  ColorBorderHover: theme.ColorPrimaryBase,
-  ColorBorderFocus: theme.ColorPrimaryBase,
-  ColorBorderDisabled: theme.ColorBorderDisabled,
-  ColorBorderDisabledChecked: theme.ColorPrimaryLight,
-  ColorBorderInvalid: theme.ColorBorderDangerBase,
-  ColorBorderInvalidHover: theme.ColorDangerDark,
-  ColorBackground: "transparent",
-  ColorBackgroundChecked: theme.ColorPrimaryBase,
-  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
-  ColorBackgroundDisabledChecked: theme.ColorPrimaryLight,
-  ColorBackgroundInvalid: theme.ColorBackgroundDangerDark,
-  ColorCheck: theme.ColorTextBodyInverse,
-  ColorTextDisabled: theme.ColorTextBody,
-  ColorTextInvalid: theme.ColorTextDanger,
-  ColorFocusOutline: theme.ColorFocusPrimary,
-  ColorFocusOutlineInvalid: theme.ColorFocusDanger,
-  FocusOutlineWidth: theme.FocusOutlineWidthSmall,
   MarginBlockEnd: theme.SpaceRemXs,
-  LabelPaddingInlineStart: theme.SpaceEmS,
-  FontSize: theme.FontSizeBody,
-  FontWeight: theme.FontWeightNormal,
+  BorderColor: theme.ColorBorderUi,
   BorderRadius: theme.BorderRadiusBase,
-  BoxSize: "1em",
   BorderSize: "2px",
   BorderStyle: theme.BorderStyleBase,
-  CheckSize: "0.75em",
   TransitionDuration: theme.TransitionDurationBase,
   TransitionTiming: theme.TransitionTimingBase,
+  FontSize: theme.FontSizeBody,
+  FontWeight: theme.FontWeightNormal,
+  HoverBorderColor: theme.ColorPrimaryBase,
+  FocusBorderColor: theme.ColorPrimaryBase,
+  FocusOutlineColor: theme.ColorFocusPrimary,
+  FocusOutlineWidth: theme.FocusOutlineWidthSmall,
+
+  BoxBackgroundColor: "transparent",
+  BoxSize: "1em",
+
+  CheckColor: theme.ColorTextBodyInverse,
+  CheckSize: "0.75em",
+
+  LabelPaddingInlineStart: theme.SpaceEmS,
+
+  CheckedBoxBackgroundColor: theme.ColorPrimaryBase,
+  CheckedBoxBorderColor: theme.ColorPrimaryBase,
+
+  DisabledBorderColor: theme.ColorBorderDisabled,
+  DisabledBoxBackgroundColor: theme.ColorBackgroundDisabled,
+  DisabledCheckedBoxBackgroundColor: theme.ColorPrimaryLight,
```

Comment:
I've been wondering how we should order tokens like this when there are multiple states or multiple variants or multiple parts all active or targeted at once. Keep it simple and use alpha order within each grouping?